test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1

```diff
@@ -130,10 +130,17 @@ Describe "Process Parent property" -Tags "CI" {
         & $powershellexe -noprofile -command '(Get-Process -Id $PID).Parent' | Should -Not -BeNullOrEmpty
     }
 
+    It "Has Parent process property in linux" -Skip:(-not $IsLinux) {
+        # Added new test to check for bug as described in issue https://github.com/PowerShell/PowerShell/issue/12908
+        $powershellexe = (Get-Process -Name "$((Get-Process -Id $PID).Name)").mainmodule.filename
+        & $powershellexe -noprofile -command "Set-Content -Path .\'t ( e ( s ) t ).txt' -Value '#!/bin/bash`r`n./$PSHOME/pwsh`r`nchmod +x ./ " + "'t ( e ( s ) t )\'"
```

Comment:
You don't and shouldn't create a new pwsh to run these commands.  Here's what I recommend changing this to (you'll need to test this as I'm typing it freehand in this edit box):

```powershell
$commandName = 't ( e ( s ) t )'

$script = @"
#!/bin/sh
pwsh -noprofile -command 'while ($true) { Start-Sleep -Seconds 10 }'
"@

Set-Content -Path testdrive:/$commandName -Value $script
chmod +x (resolve-path testdrive:/$commandName -Relative)
try {
  $p = start-process -FilePath testdrive:/$commandName -PassThru
  (Get-Process $commandName).Parent | Should -Be $pid
}
finally {
  $p | Stop-Process
}
```